src/blocktypes/containers.nix

```diff
@@ -22,7 +23,38 @@
       fragment,
       fragmentRelPath,
       target,
-    }: [
+    }: let
+      inherit (n2c.packages.${currentSystem}) skopeo-nix2container;
+      img = builtins.unsafeDiscardStringContext target.imageName;
+      tags = target.meta.tags or [(builtins.unsafeDiscardStringContext target.imageTag)];
+      copyTags = let
+        skopeo = "skopeo --insecure-policy";
+        tags' =
+          builtins.toFile "${target.name}-tags.json" (builtins.unsafeDiscardStringContext (builtins.concatStringsSep "\n" tags));
+      in ''
+        export PATH=${skopeo-nix2container}/bin:$PATH
+
+        copy_tags() {
+          local name tags source img
+          source=nix:${target}
+          tags=${tags'}
+          name=$1
```

Comment:
Since it includes the schema-ish (`docker://` / `docker-daemon` / etc), `name` might not be the right name. Maybe `sourceURI` & `targetURI`, as a better alternative?

Reply:
just went the explicit route to make it as clear as possible what's happening